src/views/pages/landing-page.twig

```diff
@@ -0,0 +1,100 @@
+{#
+| Variable     | Type                  | Description |
+|--------------|-----------------------|-------------|
+| page         | Page                  |             |
+| page.title   | string                |             |
+| page.content | string                |             |
+| page.url     | string                |             |
+| page.slug    | string                |             |
```

Comment:
add documentation for landing object, use: https://www.tablesgenerator.com/markdown_tables